UKHomeOffice/cerberus-service
# COP-10461 - Task details voyage

## Description
This PR adds the voyage section to task details page

## To Test
- Pull and run against dev
- Within `../routes/airpax/TaskDetails/TaskDetailsPage.jsx` on `line 30`, paste in the contents of the fixture file `../routes/airpax/TaskDetails/__fixtures__/taskData_AirPax_AssigneeCurrentUser.fixture.json`
- in the browser, go to `localhost:8080/airpax/tasks/DEV-DEV-20220419-001`
- Confirm that the voyage section within the third column, is rendered, similar to the screen shot below.

<img width="321" alt="image" src="https://user-images.githubusercontent.com/19333750/166802823-7fc2e04c-514b-4285-b669-40ec20a39da1.png">


## Developer Checklist

\* Required
